src/web/demo/src/main/java/org/geoserver/web/demo/MapPreviewPage.html

```diff
@@ -1,34 +1,36 @@
-<html xmlns:wicket="http://wicket.apache.org/">
-<head>
-<title><wicket:message key="page.title">Layer Preview</wicket:message></title>
-</head>
-<body>
-<wicket:extend>
-  <!-- the table component -->
-  <div wicket:id="table"></div>
-  
-  <!-- The fragment for the icon -->
-  <wicket:fragment wicket:id="iconFragment">
-    <img wicket:id="layerIcon"/>
-  </wicket:fragment>
-  
-  <!-- The fragment for the common links -->
-  <wicket:fragment wicket:id="commonLinks">
-  <div style="white-space: nowrap;"><a target="_blank" wicket:id="ol"></a>&nbsp;<a target="_blank" wicket:id="kml"></a>&nbsp;<a target="_blank" wicket:id="gml"></a></div>
-  </wicket:fragment>
-  
-  <!--  The custom drop down with optgroups -->
-  <wicket:fragment wicket:id="menuFragment">
-    <select wicket:id="menu">
-      <option><wicket:message key="selectOne">Select one</wicket:message></option>
-      <optgroup label="WMS" wicket:id="wms">
-          <option wicket:id="wmsFormats"></option>
-      </optgroup>
-      <optgroup label="WFS" wicket:id="wfs">
-           <option wicket:id="wfsFormats"></option>
-       </optgroup>
-    </select>
-  </wicket:fragment>
-</wicket:extend>
-</body>
-</html>
+<html xmlns:wicket="http://wicket.apache.org/">
```

Comment:
Full reformat again, please avoid.

Reply:
Btw, the reason why we don't want full reformats is simple, GeoServer has a long history and we use "git blame" a lot to find where some code originated while looking at code for bug fixes and new feature development purposes, a change like this is like a wall in the history. Sure, tooling allows to blame the revision before a particular change, but it's needlessly annoying.

Reply:
Hi Andrea,

I believe that the full reformat here is caused by a previous full reformat on this file in the [previous commit](https://github.com/geoserver/geoserver/commit/1d6ddea79ba5c5705dbe10100fda2616622ac4fa#diff-ea942e9262fed375f477a322e498b5c8).  Wouldn't it be better to fix this problem now instead of everyone who changes that file having the same problem?  If you mix line ending styles within the one file, you can avoid the full reformat, but then the next person would have to figure that situation out again, which would waste people's time like this file is currently wasting both our time...